crates/curp-external-api/src/conflict.rs

```diff
@@ -0,0 +1,55 @@
+#![allow(clippy::module_name_repetitions)]
+
+/// Insert into speculative pool
```

Comment:
The term 'operation' encompasses not only insertion, but also other actions. Please adjust this comment accordingly.